app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>CodeChallenge</title>
+      <%= javascript_include_tag('https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js') %>
```

Comment:
Jquery is not needed here since it will already be a part of the application with the [`jquery-rails` Gem](https://github.com/rails/jquery-rails). If you're having issues with including the `home` script, maybe try including it after the `javascript_include_tag 'application'` line.